packages/cli-core/src/package-manager-controller/package_manager_controller_base.ts

```diff
@@ -50,18 +49,6 @@ export abstract class PackageManagerControllerBase
     );
   }
 
-  /**
-   * getWelcomeMessage - returns a welcome message for the customer
-   */
-  getWelcomeMessage = () => {
```

Comment:
This method is problematic because it introduces a circular dependency between the formatter and the package manager controller. It doesn't really seem like it belonged here in the first place.

I have moved the welcome message logic to `amplify_project_creator`.

Reply:
With my latest refactor, there is no longer a circular dependency issue here but I propose we still remove this because I don't think this method belongs on this class in the first place.